### PR TITLE
Automate travis

### PR DESCRIPTION
**Pull request by vikalpj**
Some Automation for https://github.com/sroze/ngInfiniteScroll/issues/316

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,10 +2,17 @@ sudo: false
...
+  -in id_rsa_nginfinite.enc -out id_rsa_nginfinite -d
+- sh -e /etc/init.d/xvfb start
+env:
+  global:
+  - ENCRYPTION_LABEL: f635dc9b0bb1
+  - COMMIT_AUTHOR_EMAIL: ''
 before_script:
-  - "export DISPLAY=:99.0"
-  - until xwininfo -root > /dev/null 2>&1; do echo "waiting for xvfb"; done
+- export DISPLAY=:99.0
```
Comment (graingert) [inline .travis.yml]: can you restore these lines so they're not changed

File: make-bower-build.sh [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+ENCRYPTED_KEY_VAR="encrypted_${ENCRYPTION_LABEL}_key"
+ENCRYPTED_IV_VAR="encrypted_${ENCRYPTION_LABEL}_iv"
+ENCRYPTED_KEY=${!ENCRYPTED_KEY_VAR}
+ENCRYPTED_IV=${!ENCRYPTED_IV_VAR}
+openssl aes-256-cbc -K $ENCRYPTED_KEY -iv $ENCRYPTED_IV -in id_rsa_nginfinite.enc -out deploy_key -d
+chmod 600 deploy_key
+eval `ssh-agent -s`
+ssh-add deploy_key
+
+BOWER_REPO='git@github.com:sroze/ngInfiniteScroll-bower.git'
```
Comment (graingert) [inline make-bower-build.sh]: `This should go to git@github.com:ng-infinite-scroll/ng-infinite-scroll-bower.git`

File: .travis.yml [hunk 3]
```diff
@@ -2,10 +2,17 @@ sudo: false
...
+  - openssl aes-256-cbc -K $encrypted_f635dc9b0bb1_key -iv $encrypted_f635dc9b0bb1_iv
+    -in id_rsa_nginfinite.enc -out id_rsa_nginfinite -d
+  - sh -e /etc/init.d/xvfb start
+env:
+  global:
+  - ENCRYPTION_LABEL: f635dc9b0bb1
+  - COMMIT_AUTHOR_EMAIL: ''
 before_script:
-  - "export DISPLAY=:99.0"
+  - export DISPLAY=:99.0
```
Comment (graingert) [inline .travis.yml]: this line is still changed

File: make-bower-build.sh (hunk 2 above)
Comment (vikalpj) [inline make-bower-build.sh]: done

File: .travis.yml (hunk 3 above)
Comment (vikalpj) [inline .travis.yml]: its already fixed can you try refreshing the page. :) https://www.dropbox.com/s/v0j14o23ydkikoc/Screenshot%202016-10-05%2015.25.28.png?dl=0

**Pull request closed without merging**